### PR TITLE
LFR-3193: Add ContentHub Card

### PR DESCRIPTION
**Pull request by zlahham**
- Add ContentHubCard Component
- Add font-face for missing typography

Screenshot 2020-11-12 at 09 25 09
Screenshot 2020-11-12 at 09 24 38

Co-authored-by: Janie Amero 43782388+jamero102@users.noreply.github.com

### CODE REVIEW

File: packages/core/src/components/Cards/EditorialCard/ContentHubCard.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      <div className={css.imageContainer}>
+        <FittedImage src={src} alt={title} className={css.image} />
+      </div>
+      <div className={css.content}>
+        <div className={css.categoryContainer}>
+          <span className={css.category}>{category}</span>
+          {date && category && ' · '}
+          <span className={css.date}>{date}</span>
+        </div>
+        <a href={href} className={css.title}>{title}</a>
```
Comment (Denyer) [inline packages/core/src/components/Cards/EditorialCard/ContentHubCard.js:61]: Suggest we use a heading tag here:
```suggestion
        <h3><a href={href} className={css.title}>{title}</a></h3>
```

File: packages/core/src/components/Cards/EditorialCard/ContentHubCard.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        <p className={css.description}>{description}</p>
+        <div className={css.chipContainer}>
+          { tags && tags.map((tag) => (
+            <Chip
+              href={tag.href}
+              text={tag.name}
+              className={css.chip}
+            />
+          ))}
+        </div>
```
Comment (Denyer) [inline packages/core/src/components/Cards/EditorialCard/ContentHubCard.js:71]: Could we move this to a dedicate component to handle the spacing, like ChipGroup or something?

File: packages/core/src/components/Cards/EditorialCard/ContentHubCard.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  position: absolute;
+  z-index: 0;
+  width: 100%;
+  height: 100%;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  transition: opacity 200ms ease-in-out;
+  z-index: -1;
+  box-shadow: 0px 0px 5px 0px rgb(216,216,216,1), 0px 0px 15px 0px rgb(0,0,0,0.25);
```
Comment (zlahham) [inline packages/core/src/components/Cards/EditorialCard/ContentHubCard.css:21]: Can we use the variable colours for these?

**Review by zlahham — COMMENTED** [review]
Small comment/question

File: packages/core/src/components/Cards/EditorialCard/ContentHubCard.css (hunk 3 above)
Comment (jamero102) [reply packages/core/src/components/Cards/EditorialCard/ContentHubCard.css:21]: They don't match up with any of the variable colors

Comment (zlahham) [reply packages/core/src/components/Cards/EditorialCard/ContentHubCard.css:21]: oh ok 👍

**Review by jamero102 — APPROVED** [review]

**Review by zlahham — COMMENTED** [review]
I tried approving this but Github does not allow that, since I worked on it

**Review by tblakey89 — APPROVED** [review]

**Pull request merged**